src/presence_analyzer/static/presence_start_end.html

```diff
@@ -90,39 +90,41 @@
                     dropdown.show();
                     loading.hide();
                 });
+
                 $('#user_id').change(function(){
                     var selected_user = $("#user_id").val();
                     var chart_div = $('#chart_div');
```

Comment:
js variables should be camel case
jquery selectors should be proceed by $ so `$chartDiv`
use 1 var